tests/integration/aof-multi-part.tcl

```diff
@@ -0,0 +1,1065 @@
+            assert_equal v2 [$client get k2]
+            assert_equal v3 [$client get k3]
+        }
+    }
+
+    clean_aof_persistence $aof_dirpath
+
+    # Tests14: Multi Part AOF can load data when manifest add new k-v
+    create_aof_dir $aof_dirpath
+    create_aof $aof_base1_filepath {
+        append_to_aof [formatCommand set k1 v1]
+    }
+
+    create_aof $aof_incr1_filepath {
+        append_to_aof [formatCommand set k2 v2]
+    }
+
+    create_aof $aof_incr3_filepath {
+        append_to_aof [formatCommand set k3 v3]
+    }
+
+    create_aof_manifest $aof_manifest_filepath {
+        append_to_manifest "file appendonly.aof.1.base.aof seq 1 type b newkey newvalue\n"
+        append_to_manifest "file appendonly.aof.1.incr.aof seq 1 type i\n"
+        append_to_manifest "file appendonly.aof.3.incr.aof seq 3 type i\n"
+    }
+
+    start_server_aof [list dir $server_path] {
+        test "Multi Part AOF can load data when manifest add new k-v" {
+            assert_equal 1 [is_alive $srv]
+            set client [redis [dict get $srv host] [dict get $srv port] 0 $::tls]
+            wait_done_loading $client
+
+            assert_equal v1 [$client get k1]
+            assert_equal v2 [$client get k2]
+            assert_equal v3 [$client get k3]
+        }
+    }
+
+    clean_aof_persistence $aof_dirpath
+
+    # Tests15: Multi Part AOF can load data when some AOFs are empty
+    create_aof_dir $aof_dirpath
+    create_aof $aof_base1_filepath {
+        append_to_aof [formatCommand set k1 v1]
+    }
+
+    create_aof $aof_incr1_filepath {
+    }
+
+    create_aof $aof_incr3_filepath {
+        append_to_aof [formatCommand set k3 v3]
+    }
+
+    create_aof_manifest $aof_manifest_filepath {
+        append_to_manifest "file appendonly.aof.1.base.aof seq 1 type b\n"
+        append_to_manifest "file appendonly.aof.1.incr.aof seq 1 type i\n"
+        append_to_manifest "file appendonly.aof.3.incr.aof seq 3 type i\n"
+    }
+
+    start_server_aof [list dir $server_path] {
+        test "Multi Part AOF can load data when some AOFs are empty" {
+            assert_equal 1 [is_alive $srv]
+            set client [redis [dict get $srv host] [dict get $srv port] 0 $::tls]
+            wait_done_loading $client
+
+            assert_equal v1 [$client get k1]
+            assert_equal "" [$client get k2]
+            assert_equal v3 [$client get k3]
+        }
+    }
+
+    clean_aof_persistence $aof_dirpath
+
+    # Tests16: Multi Part AOF can load data from old version redis
+    create_aof $aof_old_version_filepath {
+        append_to_aof [formatCommand set k1 v1]
+        append_to_aof [formatCommand set k2 v2]
+        append_to_aof [formatCommand set k3 v3]
+    }
+
+    start_server_aof [list dir $server_path] {
+        test "Multi Part AOF can load data from old version redis" {
+            assert_equal 1 [is_alive $srv]
+
+            set client [redis [dict get $srv host] [dict get $srv port] 0 $::tls]
```

Comment:
can't we use `set client [redis_client]`?

Reply:
No, will report err:
```
Executing test client: key "host" not known in dictionary.
key "host" not known in dictionary
    while executing
"dict get $srv $property"
```